src/plugins/intel_npu/src/al/include/intel_npu/al/config/common.hpp

```diff
@@ -212,6 +212,24 @@ struct LOADED_FROM_CACHE final : OptionBase<LOADED_FROM_CACHE, bool> {
     }
 };
 
+//
+// WORKLOAD_TYPE
+//
+
+struct WORKLOAD_TYPE final : OptionBase<WORKLOAD_TYPE, ov::WorkloadType> {
```

Comment:
Should we define this in runtime.hpp since we only intend to use it in the backend?

Reply:
Done